target/zephyr/Kconfig

```diff
@@ -230,6 +247,36 @@ if MENDER_MCU_CLIENT
 
     endmenu
 
+    if MENDER_PLATFORM_MEM_TYPE_DEFAULT
+
+        menu "Memory options (ADVANCED)"
+
+            config MENDER_SEPARATE_HEAP
+                bool "Use separate heap for Mender"
+                default y
+                help
+                    Mender can either allocate dynamic memory from its own heap (default) or from the system heap.
+
+            config MENDER_HEAP_SIZE
+               int "Mender heap size (kB)"
+               depends on MENDER_SEPARATE_HEAP
+               default 10
+               range 4 32
+               help
+                   Size of the Mender heap.
+
+            config HEAP_MEM_POOL_ADD_SIZE_MENDER
```

Comment:
Unused.

Reply:
This is a trick, as explained on Slack and when demoing. I added a comment to make it clearer.